pkg/http/http_test.go

```diff
@@ -59,6 +59,7 @@ func TestMakeRequestForGet(t *testing.T) {
 }
 
 func TestMakeRequestForPostWithPayload(t *testing.T) {
+	t.Parallel()
```

Comment:
any reason to make this parallel?

Reply:
even this might not be the right temp solution as it's still throwing error here https://github.com/goto/guardian/actions/runs/12025528810/job/33522830402?pr=188